packages/components/pie-webc-core/src/test-helpers/components/web-component-test-wrapper/webComponentTestWrapper.scss

```diff
@@ -0,0 +1,19 @@
+.c-webComponentTestWrapper {
+    padding: var(--dt-spacing-c) var(--dt-spacing-e);
```

Comment:
Thoughts?
```suggestion
    padding-block: var(--dt-spacing-c);
    padding-inline: var(--dt-spacing-e);
```

Reply:
I think it's unnecessary currently as we don't need to tweak them individually. easy to refactor later on but I don't see an immediate benefit personally